specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/policyStates.json

```diff
@@ -1161,8 +1161,10 @@
           }
         }
       }
-    },
-    "/{nextLink}": {
+    }
+  },
+  "x-ms-paths": {
+    "{nextLink}?Next paging op for policy states": {
```

Comment:
@iscai-msft had changed this back to {nextLink} in the original PR (https://github.com/Azure/azure-rest-api-specs/pull/14035/commits/cf7a5ea09c6f889942dedfd6c903b8b82d97457e).  Should check with them

Reply:
@pilor I ended up having to revert that change. In `"paths"`, we have to have a leading backslash, so I had to move that `nextLink` declaration into `x-ms-paths`. To distinguish it from the other `nextLink` we have in `x-ms-paths`, I added the "query parameters"